scripts/build-ios-simulator.sh

```diff
@@ -4,4 +4,4 @@ APP_NAME="Woof"
 xcodebuild -project Woof/${APP_NAME}.xcodeproj \
            -target ${APP_NAME} \
            -configuration Release \
-           -sdk iphonesimulator16.2
+           -sdk iphonesimulator16.4
```

Comment:
As this PR is blocking others, it will be not fixed now.
The [new issue](https://github.com/ios-course/ironfoudation-team-project/issues/193) was created.